components/widgets/menu/menu_items/menu_item_action.test.jsx

```diff
@@ -46,6 +46,23 @@ describe('components/MenuItemAction', () => {
     Extra Text
   </span>
 </button>
+`);
+    });
+    test('should match snapshot with id', () => {
+        const wrapper = shallow(
+            <MenuItemActionImpl
+                onClick={jest.fn()}
+                text='Whatever'
+            />
+        );
+
+        expect(wrapper).toMatchInlineSnapshot(`
+<button
+  className="style--none"
+  onClick={[MockFunction]}
+>
+  Whatever
+</button>
```

Comment:
Non-blocking, this should be reverted since similar [snapshot](https://github.com/mattermost/mattermost-webapp/blob/31f835c3cc0c3077f3871e92a515e683f7fcb52e/components/widgets/menu/menu_items/menu_item_action.test.jsx#L10) was already present.